java/ddlogapi/DDlogRecord.java

```diff
@@ -51,7 +51,14 @@ public void release() {
         this.shared = false;
     }
 
-    private long checkHandle() {
+    /**
+     * Same as 'release', for symmetry with the DDlogCommandVector API.
+     */
+    public void dispose() {
+        this.release();
+    }
+
+    long checkHandle() {
```

Comment:
`checkHandle` dropped the private annotation. is that intentional?

Reply:
Yes, because it needs to be called by DDlogAPI, which is in the same package.